boards/arduino/nicla_vision/arduino_nicla_vision_stm32h747xx_m7.dts

```diff
@@ -192,3 +186,81 @@ zephyr_udc0: &usbotg_hs {
 	phys = <&otghs_ulpi_phy>;
 	status = "okay";
 };
+
+&i2c3 {
+	status = "okay";
+	pinctrl-0 = <&i2c3_scl_pa8 &i2c3_sda_pc9>;
+	pinctrl-names = "default";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	gc2145: gc2145@3c {
+		compatible = "gc,gc2145";
+		reg = <0x3c>;
+		status = "okay";
+
+		port {
+			gc2145_ep_out: endpoint {
+				remote-endpoint = <&dcmi_ep_in>;
+			};
+		};
+
+	};
+};
+
+&dcmi {
+	pinctrl-0 = <&dcmi_d0_pc6 &dcmi_d1_pc7 &dcmi_d2_pe0 &dcmi_d3_pe1
+		     &dcmi_d4_pe4 &dcmi_d5_pd3 &dcmi_d6_pe5 &dcmi_d7_pe6
+		     &dcmi_pixclk_pa6 &dcmi_hsync_pa4 &dcmi_vsync_pg9>;
+
+	pinctrl-names = "default";
+	status = "okay";
+
+	sensor = <&gc2145>;
```

Comment:
Just FYI, [`sensor`](https://github.com/zephyrproject-rtos/zephyr/blob/86af9c4b0766842aa97b5b9bead12c71ca5d69ed/drivers/video/video_stm32_dcmi.c#L479) could eventually be swapped for [`source`](https://github.com/zephyrproject-rtos/zephyr/pull/72623#discussion_r1598630537).
Nothing to do here now though!

Reply:
Is there any diference in that sense? Because I blindly copied this section from other examples like provided by NXP and the own binding from ST DCMI.

Reply:
Sorry, I meant that some day, the word "sensor" could be replaced by "source" *in a future pull request*.
This is just some "news" about evolution of the API. Here the driver uses "sensor", so no way to use something else for now.